### ISSUE TITLE
Missing posterior plots in sashimi plot

### ISSUE THREAD

**Issue by kwglam** [huangyh09/briekit#5]
Hi,

I am using the "samples.csv.gz" file generated by brie (splicing quantification) as a "miso_files" input for the sashimi plot. However, no posterior plot is generated. You showed in the 'https://github.com/huangyh09/briekit/blob/master/sashimi_plot/settings/sashimi_settings_brie.txt' file that it is possible to do so. Any idea why?

Gabriel

**Comment by huangyh09** [comment]
Hi Gabriel,

Thanks for the issue. Could you share the log (warnings/errors)? Can't figure it out immediately.

Yuanhua

**Comment by kwglam** [comment]
Hi Yuanhua,

Thanks for your reply. 'Posterior plot failed' is the message that I get. Sashimi plots are fine but no posterior plots on the right panel. My guess is the script that you shared in github is different from the one you used to generate Fig. 4c in the BRIE paper, https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1248-5#Abs1.
 
Any chance if you could kindly share the script again? Thanks!

Gabriel

**Comment by huangyh09** [comment]
Hi Gabriel,

Thanks for the log. It looks from this line:
https://github.com/huangyh09/briekit/blob/master/sashimi_plot/plot_utils/plot_gene.py#L289

Could you double check if the "miso_prefix" and "miso_files" are all correct? They mean the brie prefix/files here (not a good inherit I admit). If the file can't be found in L274, it will be skipped and return the above log.

This repo is already the latest version and the one we used for the figures. We also reproduced it in a protocol paper, with this tutorial: https://github.com/huangyh09/brie/blob/master/examples/gastrulation/brie_sashimi.sh#L22
But unfortunately, we didn't keep the raw bam files anymore and it needs to be downloaded again with the bash file.

Yuanhua

**Comment by kwglam** [comment]
Yes, the brie prefix/files are correct. But I should point out that the brie files are pointing to the "samples.csv.gz" files, while the miso files in the original miso 'sashimi_plot_settings.txt' file are pointing to a folder containing a list of chr folders. My guess is this is the reason why posterior plots are not working. I am wondering if you have changed the script to read 'sample.csv.gz' files in your paper instead. Thx!!

**Comment by huangyh09** [comment]
Thanks. If memory is correct, I didn't have any customised scripts for generating figures. This repo should be up to date, as also for keeping records.

Yuanhua
